src/styles/main.scss

```diff
@@ -0,0 +1,3 @@
+@import "./utils/variables.scss";
+@import "./normalize/normalize.scss";
+@import "./blocks/watch.scss";
```

Comment:
The file name 'watch.scss' should be more specific to the task, like 'stopwatch.scss', to avoid confusion. Update the name and import accordingly.